docs/relay.md

```diff
@@ -142,11 +142,7 @@ const userType = new GraphQLObjectType({
     name: {
       type: GraphQLString
     },
-    tasks: {
-      type: userTaskConnection.connectionType,
-      args: userTaskConnection.connectionArgs,
-      resolve: userTaskConnection.resolve
-    }
+    tasks: userTaskConnection.field
```

Comment:
add something that still shows the structure of the value

Reply:
i almost never use the resolver as-is, as i suspect is the case for many, so we still want to highlight how that's done.

Reply:
> i almost never use the resolver as-is
It doesn't seem like there are any examples in the documentation of using the resolver in custom ways.  I had assumed that if there were a good reason to *not* use the resolver as-is then `before` and `after` resolve hooks would be pointless.  But I can show the structure of the value